src/utils/web3-provider/providers/given-provider.js

```diff
@@ -52,6 +52,8 @@ class CustomRequestManager extends Web3RequestManager {
 class GivenProvider {
   constructor(host) {
     this.givenProvider = host;
+    if (typeof this.givenProvider === 'string')
+      throw new Error('Not supported network type');
```

Comment:
yea I dont think this is correct

Reply:
is there a way to replicate the issue?

Reply:
I usually get this issue when I access my wallet with web3 and have the tab idle
![GivenProviderString](https://user-images.githubusercontent.com/36307949/189449039-a14b7313-cc4b-4c50-b3a3-3d4537265ff1.png)